post-processing/fuzz_data_loader.py

```diff
@@ -87,6 +87,40 @@ def __init__(self, elem) -> None:
         self.total_cyclomatic_complexity: int = 0
 
 
+class BranchProfile:
+    """
+    Class for storing information about conditional branches collected by LLVM pass.
+    """
+    def __init__(self) -> None:
+        self.branch_line = str()
```

Comment:
perhaps `branch_pos` instead? point being it's more than just the line number, which I thought it was at first given the `line` name